test/common/vmimages.py

```diff
@@ -29,6 +30,8 @@
 BASE = testinfra.TEST_DIR
 IMAGES = os.path.join(BASE, "images")
 DATA = os.path.join(os.environ.get("TEST_DATA", BASE), "images")
+GIT_IMAGE_PATH = "test/images"
```

Comment:
added comment to clarify
this is the path git uses within the repo, not on the filesystem (which would be IMAGES)